src/jdk.management/windows/native/libmanagement_ext/OperatingSystemImpl.c

```diff
@@ -291,7 +291,7 @@ static const DWORD PDH_PROCESSOR_TIME_IDX = 6;
 static const DWORD PDH_PROCESS_IDX = 230;
 static const DWORD PDH_ID_PROCESS_IDX = 784;
 
-/* useful pdh fmt's */
+/* PDH format patterns, and lengths of their constant component. */
 static const char* const OBJECT_COUNTER_FMT = "\\%s\\%s";
 static const size_t OBJECT_COUNTER_FMT_LEN = 2;
```

Comment:
Pre-existing but as per earlier discussions on this, what exactly do these lengths mean??

Reply:
Yes was trying to clarify that with the comment: the "constant component", the number of fixed characters.  So allocation size is e.g. OBJECT_COUNTER_FMT_LEN (==2) to account for the two backslashes in that format, plus the lengths of the two strings, plus the terminator.

Reply:
Ah! Makes sense now. :)